msticpy/data/queries/m365d/kql_m365_hunting.yaml

```diff
@@ -509,5 +491,4 @@ sources:
         | where DownloadTime - {time_column} between (0min .. 2min)
         | project-away DeviceName1
         {add_query_items}'
-      uri: "https://github.com/microsoft/WindowsDefenderATP-Hunting-Queries/blob/master/Protection%20events/SmartScreen%20URL%20block%20ignored%20by%20user.txt"
```

Comment:
We can delete the uri element if it's causing an issue - it was an idea that was never used.

Reply:
The uri field is not the issue, it's just git messing up the details of the change. The uri field has not been modified but is now the new "last line of the file"